apidoc/Titanium/Media/Alert.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Titanium.Media.Alert
+summary: An object for playing system sounds.
+description: |
+    You can use this module to provide audible alerts. 
+
+    You can use it to play short (30 seconds or shorter) sounds. The interface does not provide level, positioning, looping, or timing control, and does not support simultaneous playback: You can play only one sound at a time. 
```

Comment:
Watch out for proper line-breaks (see other API's), so the line does not get too long.